src/EditorFeatures/CSharpTest/TypeInferrer/TypeInferrerTests.cs

```diff
@@ -1609,7 +1609,7 @@ class C
         bool x = await [|M|].ContinueWith(a => { return true; }).ContinueWith(a => { return false; });
     }
 }";
-            await TestAsync(text, "global::System.Threading.Tasks.Task<System.Boolean>");
+            await TestAsync(text, "System.Object", testPosition: false);
```

Comment:
No.  It is correct.  There is no reason for us to have inferred ```Task<bool>``` in this case.  Indeed, it could have been any type there that had a ContinueWith method on it.  The old code 'worked' because it simply assumed "hey, i'm bein assigned to a bool, ergo i must be task of bool.  But that's just specious.  it's like saying:

```string s = foo().bar```  and inferring that "foo()" must return a string.  It just has no relation